tests/roles/dataplane_adoption/defaults/main.yaml

```diff
@@ -272,3 +272,16 @@ dataplane_cr: |
           ovn_monitor_all: true
           edpm_ovn_remote_probe_interval: 60000
           edpm_ovn_ofctrl_wait_before_clear: 8000
+
+dpa_dir: "../.."
+dpa_tests_dir: "{{ dpa_dir }}/tests"
+
+# ospdo env:
+# Whether source env is OSPD Director Operator
+# i.e. https://github.com/openstack-k8s-operators/osp-director-operator
+ospdo_src: false
+# rhoso namespace
+rhoso_namespace: "openstack"
+# director operator namespace
+org_namespace: "ospdo_openstack"
```

Comment:
Ah now i see your reply that the source namespace is not necessarily OSPDO-specific. In that case i'd call it "source_namespace" or "src_namespace", we've already been using "src" abbreviation in other places of the test suite for the source environment.

But i do actually think the source namespace is specific to OSPDO. In the Director environments there is no source namespace because they don't even use OCP.

Reply:
But let's keep using `org_namespace` in all patches for now and we can rename when everything is landed.